src/lib/compress.c

```diff
@@ -75,6 +75,7 @@ __RCSID("$NetBSD: compress.c,v 1.23 2012/03/05 02:20:18 christos Exp $");
 #include <string.h>
 #include <stdlib.h>
 
+#include "../common/constants.h"
```

Comment:
@ni4 I still think that it is a bad design, because all files from includes will be installed to user's include folder.

Why we can't merge it one single file or make all files inside subfolder `rnp` for example?